performance/README.md

```diff
@@ -61,3 +61,33 @@ You can run the simulation with (in OPENWHISK_HOME)
 ```
 OPENWHISK_HOST="openwhisk.mydomain.com" CONNECTIONS="10" REQUESTS_PER_SEC="50" ./gradlew gatlingRun-ApiV1Simulation
 ```
+
+##### LatencySimulation
+
+This simulation creates actions of the following four kinds: nodejs:default, swift:default, java:default and
+python:default.
+Afterwards the action is invoked once. This is the cold-start and will not be part of the thresholds.
+Next, the action will be invoked 100 times one after each other. The last step is, that the action will be deleted.
+
+Once one language is finished, the next kind will be taken. They are not running in parallel. There are never more than
+1 activations in the system, as we only want to meassure latency of warm activations.
```

Comment:
Is this guaranteed with a blocking invoke then?

Reply:
I think you’re expecting this invariant to hold irrespective of number of controllers or invokers in system. If so can you add this stipulation as well?